src/pmpo_createTestMPMesh.cpp

```diff
@@ -173,8 +173,8 @@ MaterialPoints* initTestMPs(Mesh* mesh, int testMPOption){
         iMP += numMPsPerElement(i); 
     },numMPs);
 
-    DoubleVec3dView positions("MPpositions",numMPs);
-    DoubleVec2dView latLonPositions("MPRotLatLonPositions",numMPs); 
+    MPSView<MPF_Cur_Pos_XYZ> positions("MPpositions",numMPs);
+    MPSView<MPF_Cur_Pos_Rot_Lat_Lon> latLonPositions("MPRotLatLonPositions",numMPs); 
```

Comment:
Again, I don't understand this line: type. Need to check/discuss.

Reply:
At some point we talked about connecting the types between the mesh and material points class. So we attached the field type to the mesh field instead of hardcoding it.

Reply:
OK, I will review it again in next round.